pyk/src/pyk/kast.py

```diff
@@ -307,8 +300,9 @@ def match(self, term: KInner) -> Optional[Subst]:
         return None
 
 
-TRUE = KToken('true', BOOL)
-FALSE = KToken('false', BOOL)
+TRUE = KToken('true', 'Bool')
+FALSE = KToken('false', 'Bool')
+ktokenDots = KToken('...', 'K')
 
```

Comment:
Are these here instead of `prelude.py` because of technical reasons? They seem a better fit there.

Reply:
I see that `TRUE` is used in various places here. But the others are not. I wonder if we can avoid using `TRUE` in various places in this file...